backend/config.go

```diff
@@ -0,0 +1,76 @@
+package backend
+
+import (
+	"strings"
+
+	"github.com/grafana/grafana-plugin-sdk-go/experimental/featuretoggles"
+)
+
+const GrafanaVersion = "GF_VERSION"
+
+type Cfg struct {
+	config map[string]string
+}
+
+func NewCfg(cfg map[string]string) *Cfg {
+	// Make sure all keys are uppercase
+	normalized := make(map[string]string, len(cfg))
+	for k, v := range cfg {
+		normalized[strings.ToUpper(k)] = v
+	}
+
+	return &Cfg{config: normalized}
+}
+
+func (c *Cfg) Get(key string) string {
+	return c.config[strings.ToUpper(key)]
+}
+
+func (c *Cfg) FeatureToggles() FeatureToggles {
+	features, exists := c.config[featuretoggles.EnabledFeatures]
+	if !exists {
+		return FeatureToggles{}
+	}
+
+	fs := strings.Split(features, ",")
+	enabledFeatures := make(map[string]struct{}, len(fs))
+	for _, f := range fs {
+		enabledFeatures[f] = struct{}{}
+	}
+
+	// TODO fallback to legacy env var
```

Comment:
Not sure if we should do this or have a complete breaking change here. If we don't this API will only be usable when for plugins where Grafana is >= `10.2.0`